src/Text-Scanning/CompositionScanner.class.st

```diff
@@ -227,13 +227,13 @@ CompositionScanner >> setActualFont: aFont [
 	"Keep track of max height and ascent for auto lineheight"
 	| descent |
 	super setActualFont: aFont.
-	lineHeight == nil
+	lineHeight isNil
 		ifTrue: [descent := font descent.
-				baseline := font ascent.
-				lineHeight := baseline + descent]
+					baseline := font ascent.
+					lineHeight := baseline + descent]
 		ifFalse: [descent := lineHeight - baseline max: font descent.
-				baseline := baseline max: font ascent.
-				lineHeight := lineHeight max: baseline + descent]
+					baseline := baseline max: font ascent.
+					lineHeight := lineHeight max: baseline + descent]
```

Comment:
#ifNil:ifNotNil: can directly be used.

Reply:
While at formatting, why an increase of indent???
I would rather opt for Kent Beck square blocks...